src/ServiceFabric.Mocks/MockReliableStateManager.cs

```diff
@@ -201,6 +204,7 @@ public async Task RemoveAsync(ITransaction tx, Uri name, TimeSpan timeout)
 
         public bool TryAddStateSerializer<T>(IStateSerializer<T> stateSerializer)
         {
+            _serializers.AddSerializer(stateSerializer);
```

Comment:
To match SF implementation, this should return false if the serializer already exists in the collection